code/modules/hydroponics/plant_genes.dm

```diff
@@ -203,13 +232,30 @@
 	for(var/datum/plant_gene/trait/T in G.seed.genes)
 		T.on_slip(G, M)
 
+/datum/plant_gene/trait/slip/pod_Crossed(datum/species/pod/P,mob/living/carbon/human/H,atom/movable/AM)
+	if(!ishuman(AM))
+		return
+	var/mob/living/carbon/human/slipee = AM
+	slipee.slip(P.potency/2,H)
+
+
+/datum/plant_gene/trait/slip/pod_on_move(datum/species/pod/P, mob/living/carbon/human/H)
```

Comment:
register for the movable signal on your source mob instead

Reply:
honestly this could probably be a movement type, like FLYING, but SLIPPERY or something, that can be checked in the core slip code, instead of here.